cmd/frontend/build_frontend_env/build-frontend-env.go

```diff
@@ -0,0 +1,81 @@
+package build_frontend_env
+
+/*
+$ your_cli_tool build-frontend-env --secret "YourSecretName" --bucket "YourBucketName"
+*/
+
+import (
+    "os"
+    "strings"
+    "fmt"
+    "log"
+    "github.com/aws/aws-sdk-go/aws"
+    "github.com/aws/aws-sdk-go/aws/session"
+    "github.com/aws/aws-sdk-go/service/secretsmanager"
+    "github.com/spf13/cobra"
+    "github.com/DSGT-DLP/Deep-Learning-Playground/cli/cmd/frontend" // For frontend/
+)
+
+var secretName string // Name of the secret in AWS Secrets Manager.
+var bucketName string // Name of the bucket (Assuming one bucket here).
+
+var buildFrontendEnvCmd = &cobra.Command{
+    Use:   "build-frontend-env",
+    Short: "Build .env file for frontend/",
+    Run: func(cmd *cobra.Command, args []string) {
+        sess, err := session.NewSession(&aws.Config{Region: aws.String("us-west-2")})
+        if err != nil {
+            log.Fatal("Error creating AWS session: ", err)
+        }
+
+        smClient := secretsmanager.New(sess)
+        secretValue, err := smClient.GetSecretValue(&secretsmanager.GetSecretValueInput{SecretId: aws.String(secretName)})
+        if err != nil {
+            log.Fatal("Error retrieving secret: ", err)
+        }
+
+        path := "./frontend"
+
+        // Adding secrets to the .env file
+        writeToEnvFile(secretName, *secretValue.SecretString, path)
+
+        // ** need to add project constants to .env file **
+        writeToEnvFile("PROJECT_CONSTANT", "YourConstantValue", path)
```

Comment:
What is the purpose of this constant?

Reply:
i thought this is where our project constants go when u mentioned it in our call , please clarify if im wrong though

Reply:
I mean you're right, but isn't `YourConstantValue` some placeholder value that's not actually used?

Reply:
yea it is. should i just remove this line?

Reply:
@karkir0003

Reply:
If it's not used, remove it @codingwithsurya